server/src/pages/shot/text-tool.js

```diff
@@ -231,6 +232,17 @@ exports.TextTool = class TextTool extends React.Component {
     }
   }
 
+  onPaste(e) {
+    window.setTimeout(() => {
+      // Remove element tags, line breaks and new line seen after pasting text
+      while (this.textInput.current.firstElementChild) {
+        this.textInput.current.removeChild(this.textInput.current.firstElementChild);
+      }
+      this.textInput.current.textContent =  this.textInput.current.textContent.replace(/\r?\n|\r/g, "");
```

Comment:
eslint doesn't complain about two spaces after the equal sign in an assignment eh?

Reply:
nope, it didn't . good catch